Extension/package.json

```diff
@@ -3541,6 +3547,16 @@
                 "category": "C/C++",
                 "icon": "$(run)"
             },
+            {
+                "command": "C_Cpp.SetVSDevEnvironment",
```

Comment:
Can all these commands be unified to `VsDeveloper` instead of `VSDev`?

Reply:
sure